### DISCUSSION TITLE
Thanks rockstar

### DISCUSSION THREAD

**Discussion by BigboiKilljoy** [YimMenu/YimMenu#2138] [General]
So Rockstars gift to their players after all these years is to ban all the people who downloaded a menu to protect themself's from the other people they would never ban. wow

**Comment by alyxme** [comment]
R* don't give a damn about the fans. If they really were an adequate company and were friends with fans, they would first of all be able to limit toxic modders whos bother other players, and not those for whom it has essentially become a part of life to spend time with friends online, but they don't differentiate between them and don't care. And this is sad.

**Comment by iDominiqe** [comment]
![image](https://github.com/YimMenu/YimMenu/assets/115355282/836163ed-a88f-4d8f-804c-1a10adef6d2a)
